docs/mmteb/points/922.jsonl

```diff
@@ -0,0 +1 @@
+{"GitHub": "henilp105", "New dataset": 6}
```

Comment:
We have exceeded the deadline for dataset additions, so we have sadly stopped adding points for dataset additions (besides a few exceptions).

We would, however, welcome you to contribute to other parts of the project to obtain a co-authorship. You might get an overview as #784.

Reply:
Thanks @KennethEnevoldsen, will try contributing to other parts of the project.